src/material-experimental/mdc-chips/testing/chip-harness.ts

```diff
@@ -30,4 +32,29 @@ export class MatChipHarness extends ComponentHarness {
       exclude: '.mat-mdc-chip-avatar, .mat-mdc-chip-trailing-icon, .mat-icon'
     });
   }
+
+  /** Indicates if this chip can be deleted. */
+  async isRemovable(): Promise<boolean> {
+    const removeHarness = await this.locatorForOptional(MatChipRemoveHarness)();
```

Comment:
The presence of a remove button doesn't determine whether a chip is removable, but rather the `removable` input. Is there a use case for it, aside from throwing the error inside `remove`? If not, I think that we can get rid of this method since we don't have it on the non-MDC harness either.

Reply:
Got it, I'll remove this method.